partner_invoicing_frequency/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+===================
+Invoicing Frecuency
+===================
+
+This module allows to assign a invoicing frecuency to contacts. This information is transferred to sale order, purchase orders, pickings and invoices, but it can be freely edited.
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Only install
+
+
+Configuration
+=============
+
+To configure the stages of the opportunities that can be transferred, you need to:
+
+#. Go to Contact > Configuration > Invoicing Frecuencies
+#. Create as many invoicing frecuencies as needed.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Select a invoicing frecuency in a contact, that will be applied to ale order, purchase orders, pickings and invoices. It can be edited if necessary.
```

Comment:
```suggestion
#. Select a invoicing frecuency in a contact, that will be applied to sale orders, purchase orders, pickings and invoices. It can be edited if necessary.
```